test/test_examples_dg_multi.jl

```diff
@@ -0,0 +1,120 @@
+module TestExamplesDGMulti
+
+using Test
+using Trixi
+
+include("test_trixi.jl")
+
+# pathof(Trixi) returns /path/to/Trixi/src/Trixi.jl, dirname gives the parent directory
+EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "dg_multi")
+
+@testset "DGMulti" begin
+  @trixi_testset "Euler triangular mesh" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_weakform_2d.jl"),
+      cells_per_dimension = (4,4),
+      l2 = [0.0013463253573454783, 0.0014235911638070984, 0.0014235911638076622, 0.004721923810347086],
+      linf = [0.0015248269221803668, 0.002070690855385582, 0.002070690855385804, 0.004913338290754687]
+    )
+  end
+
+  @trixi_testset "Euler triangular mesh SBP" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_weakform_2d.jl"),
+      cells_per_dimension = (4,4),
+      approximation_type = SBP(),
+      l2 = [0.007465048853079056, 0.005297478943323888, 0.005297478943334149, 0.014701766352428895],
+      linf = [0.02149035033755564, 0.0135293837651278, 0.013529383764659286, 0.03269081948752639]
+    )
+  end
+
+  @trixi_testset "Euler quadrilateral mesh.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_weakform_2d.jl"),
+      cells_per_dimension = (4,4),
+      element_type = Quad(),
+      l2 = [0.0002909691660845978, 0.0002811425883546657, 0.0002811425883549579, 0.0010200600240538172],
+      linf = [0.0004970396373780162, 0.0004059109438805386, 0.00040591094388231497, 0.0014247618507141624]
+    )
+  end
+
+  @trixi_testset "Euler triangular mesh EC" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_weakform_2d.jl"),
+      cells_per_dimension = (4,4),
+      volume_integral = VolumeIntegralFluxDifferencing(flux_ranocha),
+      surface_integral = SurfaceIntegralWeakForm(flux_ranocha),
+      l2 = [0.008107539211003132, 0.007464472445092778, 0.007464472445093055, 0.01597648138530006],
+      linf = [0.012298218434060981, 0.013874789519390918, 0.013874789519420005, 0.040393065744379175]
+    )
+  end
+
+  @trixi_testset "Euler triangular mesh SBP EC" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_weakform_2d.jl"),
```

Comment:
Following up on the comment above, here I'd then just state the corresponding variation, e.g.,
```suggestion
  @trixi_testset "elixir_euler_weakform_2d.jl (using EC split forms with SBP)" begin
    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_weakform_2d.jl"),
```
But I realize that this might be harder to make "nice" than in other tests, so it's up to you if you want to follow these suggestions or just leave the tests as they are

Reply:
Maybe just don't use `split forms` since the EC stuff isn't using real :tm: split forms?

Reply:
All fine by me!